Software/scan_key/AuthenticationSQL.py

```diff
@@ -1,6 +1,7 @@
 from on_off import *
 from ErrorSQL import *
 import os
+import pygame
```

Comment:
was using a function from that library but instead used one from the os. So yeah we can delete that